lib/test/integration/flags.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package integration
```

Comment:
IMO the package name could simply be `e2e` ? 

imports like `lib/test/e2e/flags`

or even shorter `lib/e2e/flags`

Reply:
If we use `e2e` it will conflict with the other one in `test/e2e`

Reply:
> If we use e2e it will conflict with the other one in test/e2e

How?

if we use `test`; the import path would look like (following Daisy's suggestion)
- `lib/test/flags.go`

while the one you mentioned would be
- `test/e2e/`

These are two different paths, I dont see any conflicts.
(even if we use `e2e` it would be `lib/e2e/`; still no conflicts as the parent path is different)



We need to get this right, as immediately after merge, there would be consumer of this import path. 
Any subsequent change in this part of client repo would require changes in consumers as well.